README.md

```diff
@@ -14,21 +14,27 @@ The system is designed to run in both Kubernetes and Docker Swarm, as well as lo
 
 See [Features](https://browsertrix.cloud/features) for a high-level list of planned features.
 
-## Deployment
-
-See the [Deployment](Deployment.md) page for information on how to deploy Browsertrix Cloud.
-
 
 ## Development Status
 
 Browsertrix Cloud is currently in an alpha stage and not ready for production. This is an ambitious project and there's a lot to be done!
 
 If you would like to help in a particular way, please open an issue or reach out to us in other ways.
 
-## License
+## Documentation
 
-Browsertrix Cloud is made available under the AGPLv3 License.
+User and developer docs are located in /docs. We use [MkDocs](https://squidfunk.github.io/mkdocs-material/) with the [Material theme](https://squidfunk.github.io/mkdocs-material/).
```

Comment:
Nitpick: Is it important to mention the Material theme here, since the README is so sparse?

Reply:
Ehh, could take it or leave it.  Material for MkDocs has a bunch of flags and features that it ships with and that site is a useful reference, but the only reason this link is here is to save a Google search really.  Also I feel like people could sus out that the docs are located in `/docs` because that's pretty standard so if we're removing the link to the theme we may as well remove the rest of the line too? :P